### ISSUE TITLE
fiberassign 0.4.0 dies if a target is both a stdstar and a science target

### ISSUE THREAD

**Issue by sbailey** [desihub/fiberassign#69]
Using example files in on edison in `/scratch2/scratchdirs/sjbailey/desi/debug/fiberassign`, fiberassign/0.3.2 works fine but fiberassign/0.4.0 dies with
```
fiberassign fiberassign-params.txt
...
# map position in target list to immutable targetid at 0.0974 s
terminate called after throwing an instance of 'std::logic_error'
  what():  Duplicate targetid 4252430163699209429 in MTL
Aborted
```
The target appears in both standards-dark.fits and mtl.fits (once per file) since it is both a standard star and a requested MWS science target.  This is a perfectly legal case and should be allowed by fiberassign.

**Comment by sbailey** [comment]
Note: this problem also applies to master and the remake_fiber_positioner branch in PR #67

**Comment by sbailey** [comment]
For the record: in this sub-patch of sky, 100% of the dark time standard stars also appear as MWS targets, i.e. this is not a rare occurrence.

**Comment by sbailey** [comment]
fixed in PR #72
